transbank/webpay/webpayplus/transaction.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+from transbank.webpay.webpayplus.transaction_create_response import TransactionCreateResponse
+from transbank.webpay.webpayplus.webpayplus import *
+
+
+class Transaction:
+    CREATE_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+    COMMIT_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+    REFUND_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions/{0}/refund'
+    GET_TRANSACTION_STATUS_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions/{0}'
+
+    def __init__(self):
+        pass
+
+    @classmethod
+    def create(cls, buy_order, session_id, amount,
+               return_url, options=None):
+        commerce_code = None
+        api_key = None
+        base_url = None
+        if options is None:
+            commerce_code = WebpayPlus.commerce_code()
+            api_key = WebpayPlus.api_key()
+            base_url = WebpayPlus.integration_type_url()
+        else:
+            commerce_code = options.commerce_code
+            api_key = options.api_key
+            WebpayPlus.integration_type_url = options.integration_type
```

Comment:
Is this correct? Are we assigning a "type" of integration (which is probably a plain string) to a variable that sounds like it contains an url?

Reply:
And besides that: Are we really really sure we want to modify what looks like a global variable here?

Reply:
1. You've got the reason @leosoto, however Python offers a mechanism on which we assign values to attributes unlike as we do in java or php. In this case `options.integration_type`corresponds to a key for integration_type_url that is a dictionary and this process is to select a value according with that key

2. Python has some weird ways to manage class and instances attributes, is a class variable, we can use the term "global" because all the instances from that class will get the same value from this class var. Maybe we can discuss more in detail and help me to solve.

Reply:
Maybe I'm not following the code correctly. Let's start again with a couple of questions:

- Where is the `integration_type_url` setter defined? (I don't see it on the diff)

- Why do we we need to override the class-level integration type url? If I'm reading correctly, it means that if the user of the SDK sets a class-level (global, default) integration type and then calls this method with another integration type, then the class-level (global, default) integration type will be changed! Wouldn't that surprise you as a user of the SDK?